README.md

```diff
@@ -52,10 +56,55 @@ https://user-images.githubusercontent.com/7124846/235194069-e2d65111-1440-4f85-a
 
 **Web:**
 
-1. Navigate to the frontend web project: `create_link/frontend_web`
+1. In a separate shell, change directories to the frontend web project:
+
+   ```sh
+   cd create_link/frontend_web
+   ```
+
 2. Install dependencies and start the server.
    ```sh
    npm install
    npm run dev
    ```
-3. Visit `http://localhost:3000` in your browser and create a new link.
+
+3. At this point, the frontend should be running on port 3000 (in npm) and you should see "Local:  **http://localhost:3000**".
```

Comment:
"(in npm)" does not really make sense here in the way it does for the Docker instructions above. I would just drop that part.